deno.json

```diff
@@ -19,21 +19,29 @@
   },
   "compilerOptions": { "jsx": "react-jsx", "jsxImportSource": "preact" },
   "imports": {
+    "$fresh/": "https://deno.land/x/fresh@1.7.1/",
```

Comment:
Let's leave this. We use the branch from https://github.com/denoland/fresh/pull/2266 for route and island support within plugins.